### ISSUE TITLE
[Bug] Replace with title

### ISSUE THREAD

**Issue by RozvM** [darkreader/darkreader#9152]
# Bug Report

## Website Address

europa.eu

## Bug Description

Can't enable dark mode on the mentioned website. (Seems like it's a Firefox only problem, Chrome does not have this.)

## Steps To Reproduce

1. Open Firefox.
2. Open any europa.eu website.
3. Try to enable dark mode on the site either in _Auto_ or _On_ mode.

## Expected Behavior

The extension should work on this website normally as on others.

## Actual Behavior

The extension does not enable dark mode for some reason.

## Screenshots

![2022-06-29 22-03-48](https://user-images.githubusercontent.com/74600479/176535740-31dde34e-687e-4d69-9eb6-f55e66b372dc.gif)

## System Information

- OS: Windows 19043.1766
- Browser: Firefox 102,0
- Dark Reader Version: 4.9.52

## Additional Context

As mentioned, it seems like this is only happening on Firefox.

**Comment by Gusted** [comment]
Hi @RozvM!

Do you have by any chance `europa.eu` in your sitelist?

**Comment by RozvM** [comment]
I'm not sure, the list is long. Can I search in it somehow?

**Comment by Gusted** [comment]
> I'm not sure, the list is long. Can I search in it somehow?

Not really, you could however go to the main domain, `europa.eu` and see if dark reader has listed that site. But it's likely that another domain is conflicting with the domain and therefor not being able to enable it.
